lib/Alchemy/Phrasea/Border/Checker/AbstractChecker.php

```diff
@@ -34,6 +34,11 @@ abstract class AbstractChecker implements CheckerInterface
      */
     protected $collections = [];
 
+    /**
+     * @var \collection[]
+     */
+    protected $ignoreCollections = [];
```

Comment:
better name "compareIgnoreCollections" for vars/methods, because "collections" refers to the new record to be checked, while "compare..." refers to the records in db to match against.
btw we may use "ignoreCollections" later as an opposite to "collections", to specify the ok destination(s) as a blacklist or whitelist.